### PR TITLE
Update grant-mssp-access.md

### PR DESCRIPTION
**Pull request by mshalev**
I updated to reflect support for delegated access for MDE in M365 Security Center.
I overwrote the previous Defender for Endpoint references - TBD if to have two pages (one for MDE and the other for MDE in M365D). Since this branch doesn't overwrite the original, no harm done

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ ms.technology: mde
 
 **Applies to:**
 
-- [Microsoft Defender for Endpoint](https://go.microsoft.com/fwlink/?linkid=2154037)
+- [Microsoft Defender for Endpoint in the Microsoft 365 Security Center](https://docs.microsoft.com/en-us/microsoft-365/security/mtp/microsoft-365-security-center-mde?view=o365-worldwide) [! Replace URL with fwlink]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:28]:
```suggestion
- [Microsoft Defender for Endpoint in the Microsoft 365 Security Center](https://docs.microsoft.com/en-us/microsoft-365/security/mtp/microsoft-365-security-center-mde?view=o365-worldwide) [! Replace URL with fwlink]
```
```suggestion
- [Microsoft Defender for Endpoint in the Microsoft 365 security center](https://docs.microsoft.com/microsoft-365/security/mtp/microsoft-365-security-center-mde?view=o365-worldwide) [! Replace URL with fwlink]
```

Comment (JohanFreelancer9) [reply windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:28]: Please replace the URL as indicated.

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 2]
```diff
@@ -34,26 +34,26 @@ ms.technology: mde
 
 To implement a multi-tenant delegated access solution, take the following steps:
 
-1. Enable [role-based access control](rbac.md) in Defender for Endpoint and connect with Active Directory (AD) groups.
+1. Enable [role-based access control](rbac.md) in Defender for Endpoint in Microsoft 365 Security Center and connect with Active Directory (AD) groups.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:37]:
```suggestion
1. Enable [role-based access control](rbac.md) in Defender for Endpoint in the Microsoft 365 security center and connect with Azure Active Directory (Azure AD) groups.
```

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 3]
```diff
@@ -34,26 +34,26 @@ ms.technology: mde
...
 
 2. Configure [Governance Access Packages](https://docs.microsoft.com/azure/active-directory/governance/identity-governance-overview) for access request and provisioning.
 
 3. Manage access requests and audits in [Microsoft Myaccess](https://docs.microsoft.com/azure/active-directory/governance/entitlement-management-request-approve).
 
-## Enable role-based access controls in Microsoft Defender for Endpoint
+## Enable role-based access controls in Microsoft Defender for Endpoint in Microsoft 365 Security Center
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:43]:
```suggestion
## Enable role-based access controls in Microsoft Defender for Endpoint in the Microsoft 365 security center
```

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 4]
```diff
@@ -34,26 +34,26 @@ ms.technology: mde
...
 
 1. **Create access groups for MSSP resources in Customer AAD: Groups**
 
-    These groups will be linked to the Roles you create in Defender for Endpoint. To do so, in the customer AD tenant, create three groups. In our example approach, we create the following groups:
+    These groups will be linked to the Roles you create in Microsoft 365 Defender for Endpoint in Microsoft 365 Security Center. To do so, in the customer AD tenant, create three groups. In our example approach, we create the following groups:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:47]:
```suggestion
    These groups will be linked to the Roles you create in Microsoft 365 Defender for Endpoint in the Microsoft 365 security center. To do so, in the customer AD tenant, create three groups. In our example approach, we create the following groups:
```

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 5]
```diff
@@ -34,26 +34,26 @@ ms.technology: mde
...
     - Tier 1 Analyst 
     - Tier 2 Analyst 
     - MSSP Analyst Approvers  
 
 
-2. Create Defender for Endpoint roles for appropriate access levels in Customer Defender for Endpoint.
+2. Create Defender for Endpoint roles for appropriate access levels in Customer Microsoft 365 security center Endpoint roles & groups.
 
-    To enable RBAC in the customer Microsoft Defender Security Center, access **Settings > Permissions > Roles** and "Turn on roles", from a user account with Global Administrator or Security Administrator rights.
+    To enable RBAC in the customer Microsoft 365 Security Center, access **Permissions & roles > Endpoints roles & groups > Roles** and "Turn on roles", from a user account with Global Administrator or Security Administrator rights.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:56]:
```suggestion
    To enable RBAC in the customer Microsoft 365 security center, access **Permissions & roles** > **Endpoints roles & groups** > **Roles** and "Turn on roles" from a user account with Global Administrator or Security Administrator rights.
```

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 6]
```diff
@@ -127,7 +127,9 @@ To implement a multi-tenant delegated access solution, take the following steps:
     Example:  `https://myaccess.microsoft.com/@M365x440XXX.onmicrosoft.com#/`   
 2. Approve or deny requests in the **Approvals** section of the UI.
 
-    At this point, analyst access has been provisioned, and each analyst should be able to access the customer's Microsoft Defender Security Center: `https://securitycenter.Microsoft.com/?tid=<CustomerTenantId>`
+    At this point, analyst access has been provisioned, and each analyst should be able to access the customer's Microsoft 365 Security Center: `https://security.microsoft.com/?tid=<CustomerTenantId>` with the permissions and roles they were assigned.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:130]:
```suggestion
    At this point, analyst access has been provisioned, and each analyst should be able to access the customer's [Microsoft 365 security center](https://security.microsoft.com/?tid=<CustomerTenantId>) with the permissions and roles they were assigned.
```

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md [hunk 7]
```diff
@@ -127,7 +127,9 @@ To implement a multi-tenant delegated access solution, take the following steps:
...
+    
+**Important:** Delegated access to Microsoft Defender for Endpoint in Microsoft 365 Security Center currently allows access to a single tenant per browser window.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:132]:
```suggestion
> [!IMPORTANT]
> Delegated access to Microsoft Defender for Endpoint in the Microsoft 365 security center currently allows access to a single tenant per browser window. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md (hunk 5 above)
Comment (mshalev) [reply windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:56]: "and "Turn on roles"" above needs to be replaced with "[enable role-based access control](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/rbac)"

File: windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md (hunk 1 above)
Comment (adirdidi) [reply windows/security/threat-protection/microsoft-defender-atp/grant-mssp-access.md:28]: Please don't change this URL. Please revert this change.

**Comment by adirdidi** [comment]
What is the purpose of this change? Some of the changes are not accurate and will confuse.
We are only rolling out the new Microsoft 365 Defender experience and the existing is still available. At a later phase some of those changes will make sense, but they are not just yet.

**Comment by adirdidi** [comment]
And once they do, then we will do a deeper refresh of all pages vs. a sporadic update of a single page.

**Review by adirdidi — CHANGES_REQUESTED** [review]

**Comment by adirdidi** [comment]
Taking offline with Michael.

**Pull request closed without merging**